src/cycax/cycad/engines/simple_2d.py

```diff
@@ -15,18 +15,18 @@
 z = "z"
 
 
-class Figure:
+class Simple2D:
     """This is a class that will be used to draw pyplots of various views of a 3D object.
 
     Args:
-        part_no: This is the name of the json file that needs to be decoded.
+        name: This is the name of the json file that needs to be decoded.
         side: Thi argument will suggest which side of the object the view is of.
 
     """
 
-    def __init__(self, part_no: str, side: str) -> None:
+    def __init__(self, name: str, side: str) -> None:
```

Comment:
Id like side to have a default, TOP. And be passed in as part of the config dict. That way all the Engines has the same signature and usage. Just config differs.